terraform/aws/inventory.tmpl

```diff
@@ -6,6 +6,7 @@ all:
     cluster_ip: ${cluster_ip}
     aws_stonith_tag: ${stonith_tag}
     aws_region: ${region}
+    ebs_id_to_device_name: ${jsonencode(ebs_map)}
```

Comment:
interesting it is in `all: --> vars:` section. Considering it is id of disks for a particular VM (like to say are per VM, each of them are specific info about a specific VM) I'd expect to see them in `all: --> children: --> hana: --> hosts:`